components/automate-elasticsearch/habitat/hooks/post-run

```diff
@@ -0,0 +1,41 @@
+#!{{pkgPathFor "core/bash"}}/bin/bash
+
+exec 2>&1
+
+{{#if cfg.disable ~}}
```

Comment:
we probably also want to check if ES is configured to be a proxy node and not mess with the cluster as well